### ISSUE TITLE
koriki 0.9 some emulators dont launch

### ISSUE THREAD

**Issue by kvhagberg** [rg35xx-cfw/Koriki#25]
### What happened?

nes will not start roms.
i tried with .7z and .nes extensions

pcenginecd will not start games
.chd extension

supervision will not start games
tried .7z and .sv

atari 5200 and 7800 dont appear when roms placed in folders
wonderswan and wonderswan color dont appear

**Comment by acmeplus** [comment]
Just tested NES and is working with all the cores (fceumm, quicknes, and nestopia). Roms were in .zip but I know .nes also works

For PCEngineCD you need to select mednafen_supergrafx_libretro as core

Supervision is missing the core (potator_libretro). Will be added to the next update

Atari5200 and 7800 were not correctly defined in system/.simplemenu/section_groups/consoles.ini. You can easily fix the file:
```
[ATARI 5200]
execs = /userdata/system/.simplemenu/launchers/a5200_libretro,/userdata/system/.simplemenu/launchers/atari800_libretro 
romDirs = /userdata/roms/atari5200/,/userdata/Roms/FIFTYTWOHUNDRED/
romExts = .bin,.a52,.zip,.7z

[ATARI 7800]
execs = /userdata/system/.simplemenu/launchers/prosystem_libretro,/userdata/system/.simplemenu/launchers/stella2014_libretro                                                                                                                                                                                
romDirs = /userdata/roms/atari7800/,/userdata/Roms/SEVENTYEIGHTHUNDRED/
romExts = .bin,.a78,.zip,.7z
```

Wonderswan is also missing a core, and the path is not correct. Will fix it too in the next update.

Thanks!

**Comment by kvhagberg** [comment]
thanks for the response.

I reimaged to a new microsd and nes works now.  Balena always seesm to fail validation for me

so a few other items
sg1000 does not have the correct path in the consoles.ini, needs the sg1000 path
colecovision would not work for me until i added colecovision.rom bios file
a7800 looks like there is no core (you mention stella2014 which is fro a2600)
a5200 i can get to work after adding bios and running in retroarch, does not appear in simple menu

thanks for the great work

**Comment by kvhagberg** [comment]
i connected a wired usb ethernet and downloaded the prosystem core for a7800 and it runs in retroarch but does not appear in simplemenu

i have dreamcast running in retroarch but not appearing on simplemenu
lynx worked after installing handy core

**Comment by kvhagberg** [comment]
after some work on the section-group files i have almost all showing in simplemenu and working

**Comment by freq-mod** [comment]
On my end, Neo Geo CD section does not show in SimpleMenu.

**Comment by acmeplus** [comment]
> On my end, Neo Geo CD section does not show in SimpleMenu.

Yes, there was a typo in the path, it was neocd instead of neogeocd. You can edit ``system/.simplemenu/section_groups/consoles.ini`` and update the NEO GEO CD section like this:
```[NEO GEO CD]                 
execs = /userdata/system/.simplemenu/launchers/neocd_libretro
romDirs = /userdata/roms/neogeocd/,/userdata/Roms/NEOCD/
romExts = .zip,.chd
```

**Comment by acmeplus** [comment]
Most of these issues should have been fixed in the latest release [0.9.1](https://github.com/rg35xx-cfw/Koriki/releases/tag/koriki_rg35xx_v0.9.1)

**Comment by kvhagberg** [comment]
great thanks will grab this one

**Comment by kvhagberg** [comment]
almost everything works great
drastic seems to work great now
things i tweeked
add cue as extension for dreamcast
change pcecd to mednafen_supergrafx_libretro, the other mednafen_pce_fast_libretro didnt work in my testing
atarist has .a78 extension which is wrong, changed to ipf and 7z
msx romDirs = /userdata/roms/msx1/,userdata/roms/msx2/,userdata/roms/MSX/ 
pokemon mini needs a / in romdirs
add colecovision
[COLECOVISION]
execs = /userdata/system/.simplemenu/launchers/bluemsx_libretro
romDirs = /userdata/roms/colecovision/
romExts = .zip,.col

**Comment by acmeplus** [comment]
Thanks a lot. I’ll add those changes for the next release.
